server/handlebars/embed/views/main.hbs

```diff
@@ -9,7 +9,7 @@
     {{#unless dataEmpty}}
         {{#if (isCandidate phase)}}
         <details id="embed-report-phase-container">
-            <summary id="candidate-title"><span>Warning! Unapproved Report</span></summary>
+            <summary id="candidate-title"><span><h1>Warning! Unapproved Report</h1></span></summary>
```

Comment:
Could we make this an h4 so it matches the flow of headings in the APG website?

Reply:
This has been taken care of.